cucumberFramework/src/test/java/modules/GuestCardManagementActions.java

```diff
@@ -264,8 +268,8 @@ public static void i_get_the_of_all_dresses(String info,WebDriver driver)  {
 		
 		log.info(Arrays.deepToString(pricesDouble.toArray()));
 	

```

Comment:
i have added if Loop.Now when you will run  it,it will not go inside if loop and will print "outside if body"